lesson04.md

```diff
@@ -0,0 +1,459 @@
+<!-- .slide: id="lesson4" -->
+
+# Basic Frontend - Fall 2021
+
+Lesson 4, Thursday, 2021-09-23
+
+---
+
+### Lesson overview
+
+* `undefined` datatype
+* Operator precedence
+* Operator associativity
+* Conditional statements
+
+---
+
+### Recap
+
+Name the operators:
+
+```js
+&&
+<
+===
+||
++
+**
+```
+
+---
+
+### Recap
+
+Name the operators:
+
+```js
+&&    // logical AND
+<     // less than
+===   // strict equality
+||    // logical OR
++     // addition
+**    // exponentiation
+```
+
+---
+
+### Operators and data types
+
+| Data Type | Operators | Example |
+| ----- | --------- | ------- |
+| `number` | `+ - * / **`<br/>`=== !== < > <= >=` | `5 + 4 * 3`<br/>`7 / 2 - 2`<br/>`32 > 45` |
+| `string` | `+ === !==` | `"Pi" === "Pa"`<br/>`"Hello" + " World"` |
+| `boolean` | <code>&#124;&#124; && ! === !==</code> | `true && !false` |
+
+---
+
+### What does it print?
+
+```js
+let name = "Sevtap";
+console.log("Hello " + name);
+```
+
+Answer: It prints "Hello Sevtap" <!-- .element: class="fragment" -->
+
+---
+
+### What does it print?
```

Comment:
not sure if it's important, but in the previous lesson you pointed out that we should use "output" in place of other verbs 🤷

Reply:
Fixed :)